src/options/RendererStateSyncEnhancerOptions.ts

```diff
@@ -0,0 +1,18 @@
+export type RendererStateSyncEnhancerOptions = {
+    /**
+     * Custom function used during de-serialization of the redux store to transform the object.
+     * This function is called for each member of the object. If a member contains nested objects,
+     * the nested objects are transformed before the parent object is.
+     */
+    reviver?: (this: unknown, key: string, value: unknown) => unknown
```

Comment:
I got a little confused about terminology used here. Could we use the more generally accepted "deserializer"?